hack/check-commit-message.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -o nounset
+
+commit_file=${1}
+commit_message="$(cat ${commit_file})"
+valid_commit_regex='^(NO-ISSUE|Merge|Revert|((Bug |[A-Z]+-|#)[0-9]+:))'
+
+error_msg="""Aborting commit: ${commit_message}
```

Comment:
I don't think """ has any meaning in bash. IIRC, this just means the following in python:

    error_msg = "" + """"Aborting commit: ${commit_message}

A single quote would be enough for multiline:
```suggestion
error_msg="Aborting commit: ${commit_message}
```